symplyphysics/laws/chemistry/molar_mass_via_mass_of_single_particle.py

```diff
@@ -0,0 +1,53 @@
+from sympy import Eq, solve, Symbol as SymSymbol
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    symbols,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.chemistry import (
+    atomic_weight_from_mass_mole_count as molar_mass_law,
+    avogadro_number_from_mole_count as avogadro_law,
+)
+
+# Description
+## Molar mass of a substance is the mass of 1 mole of particles that it is comprised of.
+
+molar_mass = Symbol("molas_mass", units.mass / units.amount_of_substance)
+particle_mass = Symbol("particle_mass", units.mass)
+
+law = Eq(molar_mass, particle_mass * units.avogadro)
+
+# Derive from another definition of molar mass
+
+_number_of_particles = SymSymbol("number_of_particles", integer=True)
+
+_amount_of_substance = solve(
+    avogadro_law.law, avogadro_law.mole_count
+)[0].subs(
+    avogadro_law.particles_count, _number_of_particles
+)
+
+_total_mass = _number_of_particles * particle_mass
```

Comment:
I wonder if we can derive it from https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/mass_fraction.py
or from
https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/conservation/mixture_mass_equal_sum_of_components_masses.py

Reply:
Yes, it is possible